public/app/features/explore/TraceView/components/TraceTimelineViewer/SpanDetail/index.tsx

```diff
@@ -197,22 +198,23 @@ export default function SpanDetail(props: SpanDetailProps) {
   let logLinkButton: JSX.Element | undefined = undefined;
   if (createSpanLink) {
     const links = createSpanLink(span);
-    if (links?.logLinks) {
+    const logLinks = links?.filter((link) => link.type === SpanLinkType.Logs);
```

Comment:
@aocenas This is the place where there's a button just for the first log link so we use the type here.